### PR TITLE
fix: add http checks

### PR DESCRIPTION
**Pull request by chasers**
Adds http checks which will hopefully reduce error responses during rolling deploys.

### CODE REVIEW

**Review by abc3 — APPROVED** [review]

File: deploy/fly/prod.toml [hunk 1]
```diff
@@ -40,3 +39,15 @@ processes = []
...
     restart_limit = 6
     timeout = "2s"
+  
+  [[services.http_checks]]
+    interval = 10000
+    grace_period = "5s"
+    method = "get"
+    path = "/"
+    protocol = "http"
+    restart_limit = 0
```
Comment (w3b6x9) [inline deploy/fly/prod.toml:49]: @chasers wouldn't it be better to set this to a positive number so `n` consecutive failing health checks will trigger a restart?

File: deploy/fly/prod.toml [hunk 2]
```diff
@@ -40,3 +39,15 @@ processes = []
     interval = "15s"
     restart_limit = 6
     timeout = "2s"
+  
+  [[services.http_checks]]
+    interval = 10000
+    grace_period = "5s"
+    method = "get"
+    path = "/"
+    protocol = "http"
```
Comment (w3b6x9) [inline deploy/fly/prod.toml:48]: @chasers we should set this to `https` if we're going to set `tls_skip_verify` to `false`

File: deploy/fly/prod.toml (hunk 1 above)
Comment (chasers) [reply deploy/fly/prod.toml:49]: Eventually but I don't want to do that now so we change change any behavior of the cluster after a node is up and running.

File: deploy/fly/prod.toml (hunk 2 above)
Comment (chasers) [reply deploy/fly/prod.toml:48]: "When true (and using HTTPS protocol) skip verifying the certificates sent by the server."

Seems to me that this should be `true` when using `https`.

Comment (chasers) [reply deploy/fly/prod.toml:48]: And I'm thinking these checks happen internally so unless we actually serve a cert from the instance we need to do http.  Although not sure if they're terminating https at the load balancer or what.

Comment (chasers) [reply deploy/fly/prod.toml:48]: Gonna give it a go and see what happens.

**Pull request merged**